src/raftstore/store/util.rs

```diff
@@ -544,15 +547,35 @@ pub fn get_region_approximate_split_keys_cf(
     if keys.len() == 1 {
         return Ok(vec![]);
     }
+    if keys.is_empty() || total_size == 0 || split_size == 0 {
```

Comment:
Maybe just log when `n == 0` and combine all the information there?

Reply:
Divided by zero may cause panic too, so we should check it before we can know whether `n == 0`